development/image-url-helper/pkg/promote/promote.go

```diff
@@ -103,6 +113,15 @@ func GetWalkFunc(ResourcesDirectoryClean, targetContainerRegistry, targetTag str
 	}
 }
 
+func isFileExcluded(ResourcesDirectoryClean, path string, excludes []string) bool {
```

Comment:
Use map to hold excluded files. With list you have to iterate over whole list for every values.yaml. A map better fit to the usage of data structure.

Reply:
excludes returned by the `ParseExcludes` function are now in a map